data/json/itemgroups/guns.json

```diff
@@ -207,6 +208,7 @@
       [ "ak47", 100 ],
       [ "ak74", 60 ],
       [ "an94", 40 ],
+      [ "bh_m89", 60],
```

Comment:
Too common, should be 20 (half as common as steyr aug). The flintlock values need reducing by about 60 points each and that could also be done here